k8s/src/main/scala/io/buoyant/k8s/SerializationModule.scala

```diff
@@ -0,0 +1,10 @@
+package io.buoyant.k8s
+
+import com.fasterxml.jackson.databind.module.SimpleModule
+
+/**
+ * Class used to provide LoadService-compatible serialization/deserialization plugins to k8s JSON parsing.
+ */
+abstract class SerializationModule {
```

Comment:
Not at the moment - they're just case classes, which can be handled by Jackson without any special-casing. The reason to make this pluggable is that the `ObjectMapper` is constructed in the `k8s` project, but only `namerd-storage-k8s` needs it to be able to parse `Path` and `NameTree[Path]` instances.